ukpa/templates/ukpa/_nomination_form.html

```diff
@@ -0,0 +1,31 @@
+{% load bootstrap3 %}
+{% load avatar_tags %}
+<p>Before submitting this form, you should ensure that you have read the <a href="http://2017.pyconuk.org/ukpa/trustess/">Trustee Guidelines</a> on the main conference website.
+<form method="post">
+  {% csrf_token %}
+
+  <table class="table borderless">
+    <tr>
+      <td>{% avatar user %}</td>
```

Comment:
If I don't have a Gravatar associated with my email address, this shows

![image](https://user-images.githubusercontent.com/28734/28372071-a461ff40-6c96-11e7-9db6-a77719a66762.png)

Can we change it so that it shows nothing, with a prompt to use Gravatar?  Or do you think this is overkill?